omero/apps/index.html

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: OMERO Training
```

Comment:
Why this title?

Reply:
Because I copied and pasted from the training page to start the apps page and never renamed the title.